extractor/standalone/windows/dismpatch/dismparser/dism_parser_test.go

```diff
@@ -116,7 +116,7 @@ func TestParseError(t *testing.T) {
 	}
 
 	_, _, err = Parse(string(content))
-	if err == nil || err.Error() != "Could not parse DISM output successfully" {
-		t.Errorf("Parse: Want: %v, Got: %v", "Could not parse DISM output successfully", err)
+	if err == nil || err.Error() != "could not parse DISM output successfully" {
+		t.Errorf("Parse: Want: %v, Got: %v", "could not parse DISM output successfully", err)
```

Comment:
I've opened https://github.com/google/osv-scalibr/pull/602 to replace this with an `errors.Is` check